src/pyhf/optimize/mixins.py

```
@@ -68,15 +68,40 @@ def _internal_minimize(
             raise exceptions.FailedMinimization(result) from exc
         return result
 
-    def _internal_postprocess(self, fitresult, stitch_pars, return_uncertainties=False):
+    def _internal_postprocess(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_internal_postprocess`

(ANN202)